dao/src/main/kotlin/tables/runs/repository/PackageCurationDataTable.kt

```diff
@@ -68,7 +68,7 @@ class PackageCurationDataDao(id: EntityID<Long>) : LongEntity(id) {
                 it.binaryArtifact?.mapToModel() == data.binaryArtifact &&
                         it.sourceArtifact?.mapToModel() == data.sourceArtifact &&
                         it.vcsInfoCurationData?.mapToModel() == data.vcs &&
-                        it.takeIf { it.hasAuthors }?.authors?.map { author -> author.name }?.toSet() == data.authors &&
+                        it.hasAuthors && it.authors.mapTo(mutableSetOf()) { author -> author.name } == data.authors &&
```

Comment:
This changes semantics if `it.hasAuthors == false`, previously this line could evaluate to `true` if `data.authors == null`.

Reply:
Good catch, I missed that `data.authors` is nullable.